### PR TITLE
create sawhuang.md

### PR DESCRIPTION
**Pull request by sawhuang**
https://rawgit.com/sawhuang/sawhuang.github.io/add-sawhuang-profile/#!pages/vi/profiles/sawhuang.md

### CODE REVIEW

**Comment by singharpita** [comment]
https://raw.githack.com/sawhuang/sawhuang.github.io/add-sawhuang-profile/#!pages/vi/profiles/sawhuang.md

File: pages/vi/profiles/sawhuang.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+My name is **Sawyer Huang**, and I am a freshman at [Cornell University](https://www.cornell.edu/) studying [Computer Science](https://www.cs.cornell.edu/).
+I'm from a small town called Gardiner, NY, but I currently reside in Ithaca, NY as a full-time student.
+When I'm not programming, you can probably find me at the gym lifting weights.
+
+I am excited because I believe this internship is a great opportunity to gain exposure to modern professional coding practices.
+I want to serve as a reliable team member on all endeavors throughout my time at OLE.
+
+Location (hometown)      | Timezone      | OS  
+:------------- |:-------------:| -----:
```
Comment (singharpita) [inline pages/vi/profiles/sawhuang.md]: markdown use is good but alignment is causing things not in place. (Inconsistent layout)

File: pages/vi/profiles/sawhuang.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Who is the man behind the curtain?
```
Comment (singharpita) [inline pages/vi/profiles/sawhuang.md]: These header tags also create sidebar and it does not look nice with these content as sidebar menu. So please change it to look better.

**Comment by lmmrssa** [comment]
@sawhuang Purpose of this step is to help you learn different markdown. But saying that you should also try to make sure the changes you make looks good when you look on the page.

**Comment by lmmrssa** [comment]
@sawhuang @dogi Closing this PR as it seems to have been open for long time and no update on it.

**Pull request closed without merging**